big_tests/tests/mongoose_push_mock.erl

```diff
@@ -34,7 +34,7 @@ wait_for_push_request(Token) ->
         {push_request, Token, Body} ->
             Body
     after 10000 ->
-              ct:error(timeout_waiting_for_push_request)
```

Comment:
I think we need to get back to the `ct:error`, right?

Reply:
Shouldn't we change it to `ct:fail/1` ?

Reply:
What's the difference between `ct:fail` and `ct:error`?

Reply:
Other than it doesn't exist :) Yes, `ct:fail` will work here.